app/controllers/titles_controller.rb

```diff
@@ -33,4 +49,21 @@ def set_title
   def titles
     Title.configure(config)
   end
+
+  def combined_resource
+    json = JSON.parse request.body.read
```

Comment:
```

Reply:
and i remember you mentioned something about string keys, and you weren't sure why it was being enforced.  usually here we'd be dealing with `ActionController::Parameters` which implement "indifferent access" (string keys and symbol keys resolve identically.

Reply:
Added the `rescue` to the `#create` portion so we don't end up with a double `render`.